test/unittests/t_uuid.cc

```diff
@@ -62,7 +62,7 @@ TEST(T_Uuid, FailRead) {
   fclose(f);
   UnlinkGuard unlink_guard(path);
   UniquePtr<Uuid> uuid(Uuid::Create(path));
-  EXPECT_TRUE(uuid == NULL);
+  EXPECT_TRUE(uuid.IsValid());
```

Comment:
Ah, that's important.  We should be confident that the test fails with the reversed condition.  Can follow it up?